scripts/ci/docker-compose/integration-statsd.yml

```diff
@@ -19,15 +19,37 @@ version: "3.7"
 services:
   statsd-exporter:
     image: apache/airflow:airflow-statsd-exporter-2020.09.05-v0.17.0
+    container_name: "breeze-statsd-exporter"
     ports:
       - "9125:9125"
       - "9125:9125/udp"
       - "29102:9102"
 
+  prometheus:
+    image: prom/prometheus
+    container_name: "breeze-prometheus"
+    user: "0"
+    ports:
+      - "29090:9090"
+    volumes:
+      - ./prometheus.yml:/etc/prometheus/prometheus.yml
+      - ./prometheus/volume:/prometheus
+
   grafana:
     image: grafana/grafana:8.2.4
+    container_name: "breeze-grafana"
+    environment:
+      GF_AUTH_ANONYMOUS_ENABLED: true
+      GF_AUTH_ANONYMOUS_ORG_NAME: "Main Org."
+      GF_AUTH_ANONYMOUS_ORG_ROLE: "Admin"
```

Comment:
Not a good idea in production, but this configures Grafana to auto-login as an admin.  Seems fine to me since this is a test/dev environment.

Reply:
We have plenty of admin/admin , airflow/airflow in those files already :).

Reply:
"Main Org." is the default org and only available option without also configuring the organization/team/user structure which felt excessive in this case.